Dockerfile

```diff
@@ -81,7 +81,11 @@ RUN echo "source activate gatk" > /root/run_unit_tests.sh && \
 
 RUN cp -r /root/run_unit_tests.sh /gatk && \
     cp -r /root/gatk.jar /gatk
-ENV CLASSPATH=/gatk/gatk.jar:$CLASSPATH PATH=$CONDA_PATH/envs/gatk/bin:$CONDA_PATH/bin:$PATH
+ENV CLASSPATH=/gatk/gatk.jar:$CLASSPATH IN_GATKLITE_DOCKER=true PATH=/gatk:$PATH
+
+FROM gatk-lite AS gatk
+
+ENV PATH=$CONDA_PATH/envs/gatk/bin:$CONDA_PATH/bin:$PATH IN_GATKLITE_DOCKER=
```

Comment:
I hate the trailing nothing in the assignment, is that really how it works?

Reply:
This is really how it works, unfortunately.  There's not other way to unset it

Reply:
At least not that I've been able to find